packages/remix-server-runtime/__tests__/server-test.ts

```diff
@@ -612,6 +612,35 @@ describe("shared server runtime", () => {
       expect(indexLoader.mock.calls.length).toBe(1);
     });
 
+    test("data request calls loader returning raw fetch response", async () => {
+      let build = mockServerBuild({
+        root: {
+          default: {},
+        },
+        "routes/_index": {
+          parentId: "root",
+          index: true,
+          loader() {
+            return fetch(`https://remix.run/docs/en/main?_data=root`);
```

Comment:
I don't really want to keep this here but the only way to replicate this issue was to actually get a legit `Response` from a `fetch` call which is why we didn't run into this before.  Thoughts on how to best prevent a regression in this area without spamming our prod server?